sub-logs/farish.json

```diff
@@ -0,0 +1,17 @@
+{
+  "repo": "https://github.com/iamfarishirfan/website",
+
+  "owner": {
+    "username": "iamfarishirfan",
+    "email": "hello.farish.irfan@gmail.com"
+  },
+
+  "target": {
+    "CNAME": {
+      "name": "farish",
+      "value": "700b3eafa2ce95949fe9e0e2._acme.deno.dev"
```

Comment:
I'm unable to load the CNAME record, as it's doesn't load.

Reply:
I'm just following the instructions provided by my provider. Can you merge the PR first and if it doesn't work, at least I have proof it doesn't work which I can show to him.